### PR TITLE
fix: allow for default props to circular progress component

### PR DESCRIPTION
**Pull request by chrisolsen**
DDI-765: add docs for the new default props

### CODE REVIEW

File: libs/docs/src/components/common/CircularProgress.stories.mdx [hunk 1]
```diff
@@ -91,6 +91,30 @@ export const FullscreenProgressTemplate = () => {
...
+
+<Tabs>
+  <Tab label="Web Component">
+    <CodeSnippet lang="html" code={`
+      <goa-circular-progress visible />
+    `}/>
+  </Tab>
+  <Tab label="Angular">
+    <CodeSnippet lang="html" code={`
+      <goa-circular-progress (visible)="visible"></goa-circular-progress>
```
Comment (bluekurinji) [inline libs/docs/src/components/common/CircularProgress.stories.mdx:108]: This should be 
`<goa-circular-progress [visible]="visible"></goa-circular-progress>`

**Comment by bluekurinji** [comment]
Please refer the ticket number in the commit :-)

**Review by bluekurinji — APPROVED** [review]

File: libs/docs/src/components/common/CircularProgress.stories.mdx (hunk 1 above)
Comment (bluekurinji) [reply libs/docs/src/components/common/CircularProgress.stories.mdx:108]: missed the above change request and approved it by mistake

**Review by bluekurinji — CHANGES_REQUESTED** [review]

**Comment by chrisolsen** [comment]
> [DDI-765](https://goa-dio.atlassian.net/browse/DDIDS-765): add docs for the new default props

The storybook issue is referenced in the commit message, just not the first line for reason that this PR is mainly adding default values, which previously did not exist.

**Review by bluekurinji — APPROVED** [review]

**Pull request merged**